gis/mapnik/Portfile

```diff
@@ -6,11 +6,9 @@ PortGroup           boost 1.0
 
 name                mapnik
 version             3.1.0
-revision            8
+revision            9
 categories          gis devel
-platforms           darwin
 license             LGPL-2.1
-supported_archs     i386 x86_64
```

Comment:
Thanks for the port history. Since then, a lot have changed. If indeed `mapnik` will not work and isn't "fixable" for ppc, then we could "unsupport" it again. At least we will know for sure.